addons/medical/scripts/Game/ACE_Medical/Components/Gadgets/SCR_ConsumableEffectBase.c

```diff
@@ -10,5 +10,6 @@ modded enum SCR_EConsumableType
 modded enum SCR_EConsumableFailReason
 {
 	ACE_MEDICAL_NOT_INCAPACITATED,
-	ACE_MEDICAL_TOO_DAMAGED
+	ACE_MEDICAL_TOO_DAMAGED,
+	ACE_MEDICAL_CRITICALLY_INJURED
```

Comment:
To make it consistent with the new localization
```suggestion
	ACE_MEDICAL_TOO_DAMAGED_MEDICAL_FACILITY_REQUIRED
```

Reply:
Missed updating this variable, it's been updated to the recommended name.